packages/compass-components/src/components/form-field-container.tsx

```diff
@@ -4,6 +4,10 @@ import React from 'react';
 
 const formFieldContainerStyles = css({
   margin: `${spacing[4]}px 0`,
+  // This is to prevent buttons from other form fields being positioned on top
+  // of a select dropdown menu in this form field when the window loses focus.
```

Comment:
Is/was this a LeafyGreen issue? Might be worth seeing if the new versions (if we aren't on newest) behave the same. Looks like an e2e test is failing. Changes are fine by me. lgtm once green

Reply:
I don't know.